src/mobile/src/main/java/tupperdate/android/ui/home/profile/ProfileGeneral.kt

```diff
@@ -0,0 +1,188 @@
+package tupperdate.android.ui.home.profile
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import dev.chrisbanes.accompanist.coil.CoilImage
+import tupperdate.android.R
+import tupperdate.android.data.features.recipe.Recipe
+import tupperdate.android.ui.theme.TupperdateTheme
+import tupperdate.android.ui.theme.components.ProfilePicture
+import tupperdate.android.ui.theme.material.BrandedButton
+import tupperdate.android.ui.theme.modifier.shade
+
+@Composable
+fun ProfileGeneral(
+    name: String,
+    email: String,
+    image: Any,
+    location: String,
+    userRecipes: List<Recipe>,
+    onEditClick: () -> Unit,
+    onLocationChange: (String) -> Unit,
+    onNewRecipeClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    //TODO add the new appbar done by G-L for the new navigation
```

Comment:
The appbar will not be displayed here. You can check out the code in `Home.kt` to see how it is done.